react/PWAContext.js

```diff
@@ -54,14 +69,17 @@ const PWAProvider = ({ rootPath, children, data = {} }) => {
   const context = useMemo(() => {
     if (pwaSettings) {
       const { disablePrompt, promptOnCustomEvent } = pwaSettings
+      const isIOS = navigator && !!navigator.userAgent.match(/(iPod|iPhone|iPad)/)
```

Comment:
What is special about iOS?

Reply:
iOS devices don't have web app installation features.

Reply:
Maybe add a comment to explain